cpp/ql/lib/semmle/code/cpp/PrintAST.qll

```diff
@@ -305,14 +310,14 @@ class ExprNode extends AstNode {
 
   ExprNode() { expr = ast }
 
-  override AstNode getChildInternal(int childIndex) {
-    result.getAst() = expr.getChild(childIndex)
+  override PrintAstNode getChildInternal(int childIndex) {
+    result.(AstNode).getAst() = expr.getChild(childIndex)
```

Comment:
Nit: Consider adding a predicate`PrintAst::getAst() { none() }`.

Reply:
Left as-is.